Packs/PaloAltoNetworks_IoT/Scripts/iot_get_raci/iot_get_raci.py

```diff
@@ -30,11 +30,7 @@ def get_raci(args):
     iot_config_list_name = args.get('iot_config_list_name', 'IOT_CONFIG')
     config = get_iot_config(iot_config_list_name)
     if config is None:
-        return CommandResults(
-            outputs_prefix='PaloAltoNetworksIoT.RACI',
-            outputs_key_field="",
-            outputs=None
-        )
+        return None
```

Comment:
Don't you want to at least output some verbal message to the user in case of `None`? Maybe something like no results was found?

Reply:
I think this is okay if IOT_CONFIG is not defined in the List, and we return None here.